src/components/GameView/CardListSortable.vue

```diff
@@ -84,19 +74,13 @@ export default {
   },
   computed: {
     sortedCards() {
-      return this.sortByRank ? sortBy(this.cards, 'rank') : [...this.cards];
+      return this.sortByRank ? orderBy(this.cards, ['rank', 'suit']) : [...this.cards]; // Changed to use orderBy
     },
-    // Used to dynamically generate data-cy selectors for each card
     dataSelectorName() {
       return `data-${this.dataSelectorPrefix}-card`;
     },
   },
   methods: {
-    /**
-     * Format data selector for a specific card into object for v-bind
-     * @example result: {'data-scrap-dialog-card': '1-3'}
-     * ^^^ the ace of spades in the scrap dialog
-     **/
```

Comment:
Same as above; I think the particular formatting requirements of these stings warrant a comment to explain how they look and should be used. Again, I think it's helpful to discuss these things because in theory comments like this are more for helping new contributors than veteran maintainers, so your input is useful! Why did you choose to remove it? Do you think the code is easier to understand without this comment?

Reply:
Hi @itsalaidbacklife Thank you for reviewing the code, I have went through all the requested changes and will make the changes and update it.
For the comments, as I told you this is my first contribution and I was very excited, I have deleted the comments in haste  but will take care from next time and add the necessary comments where ever required, I understand the importance of comments these were really helpful for me to go through the code.

Reply:
Okay no problem at all. This is why we do code review. Just glad to have gotten on the same page about it